src/universal/components/RetroSidebarDiscussSection.js

```diff
@@ -51,7 +51,8 @@ const RetroSidebarDiscussSection = (props: Props) => {
         const navState = {
           isActive: localStage.localStageId === stage.id, // the local user is at this stage
           isComplete: stage.isComplete, // this stage is complete
-          isDisabled: !getIsNavigable(false, phases, stage.id),
+          // isDisabled: !getIsNavigable(false, phases, stage.id),
```

Comment:
@mattkrick `getIsNavigable` causes an error, I think because it is trying to map over `phases` but phases may just be `[]` or not have any stages. Can you double check? Otherwise I think this PR is probably ready by rebasing to master and getting merged behind #2027

Reply:
@mattkrick here’s the issue I mentioned